loopchain/blockchain/blockchain.py

```diff
@@ -610,20 +645,25 @@ def confirm_block(self, confirmed_block_hash: Hash32):
                     return
 
             except KeyError:
-                if self.last_block.header.hash == confirmed_block_hash:
-                    logging.warning(f"Already added block hash({confirmed_block_hash.hex()})")
+                if self.last_block.header.hash == current_block.header.prev_hash:
+                    logging.warning(f"Already added block hash({current_block.header.prev_hash.hex()})")
                     return
                 else:
-                    except_msg = f"there is no unconfirmed block in this peer block_hash({confirmed_block_hash.hex()})"
+                    except_msg = \
+                        f"there is no unconfirmed block in this peer " \
+                        f"block_hash({current_block.header.prev_hash.hex()})"
```

Comment:
```suggestion
                        except_msg = ("there is no unconfirmed block in this peer "
                                      "block_hash({current_block.header.prev_hash.hex()})")
```
I suggest this if need break new line.

Reply:
before
```
except_msg = f"there is no unconfirmed block in this peer block_hash({confirmed_block_hash.hex()})"
```

after
```suggestion
except_msg = ("there is no unconfirmed block in this peer "
               f"block_hash({current_block.header.prev_hash.hex()}))"
```

My suggestion has to replaced with 3 lines begin from `except_msg`.